FruitFruit/FruitFruit/Controllers/BottomSheetViewController.swift

```diff
@@ -0,0 +1,178 @@
+        lblNumber.leadingAnchor.constraint(equalTo: bottomSheetView.leadingAnchor, constant: 179).isActive = true
+        
+        btnPlus.setImage(UIImage(named: "Fruitfruit_btn_Plus"), for: .normal)
+        self.view.addSubview(btnPlus)
+        btnPlus.topAnchor.constraint(equalTo: bottomSheetView.topAnchor, constant: 72).isActive = true
+        btnPlus.leadingAnchor.constraint(equalTo: bottomSheetView.leadingAnchor, constant: 265).isActive = true
+        btnPlus.addTarget(self, action: #selector(onTapPlus), for: .touchUpInside)
+        
+        btnMinus.setImage(UIImage(named: "Fruitfruit_btn_Minus"), for: .normal)
+        self.view.addSubview(btnMinus)
+        btnMinus.topAnchor.constraint(equalTo: bottomSheetView.topAnchor, constant: 72).isActive = true
+        btnMinus.leadingAnchor.constraint(equalTo: bottomSheetView.leadingAnchor, constant: 110).isActive = true
+        btnMinus.addTarget(self, action: #selector(onTapMinus), for: .touchUpInside)
+        
+        self.view.addSubview(checkOrderButton)
+        checkOrderButton.topAnchor.constraint(equalTo: bottomSheetView.topAnchor, constant: 152).isActive = true
+        checkOrderButton.leadingAnchor.constraint(equalTo: bottomSheetView.leadingAnchor, constant: 24).isActive = true
+        checkOrderButton.trailingAnchor.constraint(equalTo: bottomSheetView.trailingAnchor, constant: -24).isActive = true
+        checkOrderButton.bottomAnchor.constraint(equalTo: bottomSheetView.bottomAnchor, constant: -40).isActive = true
+        checkOrderButton.frame.size = CGSize(width: view.bounds.size.width - 48, height: 58)
+        let gradient = checkOrderButton.applyButtonGradient(colors: Constants.FruitfruitColors.buttonGradient)
+        checkOrderButton.setTitle(costcountCheckText(), for: .normal)
+        checkOrderButton.backgroundColor = .blue
+        checkOrderButton.titleLabel?.font = UIFont.preferredFont(for: .headline, weight: .semibold)
+        checkOrderButton.layer.cornerRadius = 16
+        checkOrderButton.layer.borderWidth = 1
+        checkOrderButton.layer.borderColor = UIColor(named: Constants.FruitfruitColors.button1)?.cgColor
+        checkOrderButton.layer.insertSublayer(gradient, at: 0)
+        checkOrderButton.addTarget(self, action: #selector(onTapOrder), for: .touchUpInside)
+    }
+    private func setCount() -> String {
+        let count = String(Number) + "개"
+        return count
+    }
+    private func setCost() -> String {
+        let cost = "900원 "
+        return cost
+    }
+    private func costcountCheckText() -> String {
+        return setCost() + setCount() + " 주문하기"
+    }
+    
+    override func viewDidLoad() {
+          super.viewDidLoad()
+          setupUI()
+        let dimmedTap = UITapGestureRecognizer(target: self, action: #selector(dimmedViewTapped(_:)))
+            dimmedView.addGestureRecognizer(dimmedTap)
+            dimmedView.isUserInteractionEnabled = true
+      }
+    @objc
+    func onTapPlus() {
```

Comment:
```suggestion
    func didTapPlusButton() {
```

method 는 동사로 시작되어야 호출하는 코드를 읽을 때, method를 호출하는구나 ! 라고 파악하기 쉽다고 생각합니다

Reply:
너무 감사합니다. 참고해서 수정하고 공부할게요!!